task/reindex.go

```diff
@@ -290,7 +290,14 @@ func transformZebedeeDoc(ctx context.Context, tracker *Tracker, errChan chan err
 		exporterEventData := extractorModels.MapZebedeeDataToSearchDataImport(zebedeeData, -1)
 		importerEventData := convertToSearchDataModel(exporterEventData)
 		if topicTaggingEnabled {
+			beforeTaggingTopicsCount := len(importerEventData.Topics)
 			importerEventData = tagImportDataTopics(topicsMap, importerEventData)
+			afterTaggingTopicsCount := len(importerEventData.Topics)
+			if beforeTaggingTopicsCount == afterTaggingTopicsCount {
+				tracker.Inc("docs-not-tagged")
```

Comment:
`docs-topic-untagged`?

Reply:
should we log the URI when it's not tagging?

Reply:
can do